infrastructure/cdn-in-a-box/edge/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why split these up?

Reply:
Added in efa7eac1e. I blame WebStorm

Reply:
I think you responded to the wrong comment

Reply:
So I did. Response to the right comment added in [discussion_r473302501](https://github.com/apache/trafficcontrol/pull/4972#discussion_r473302501).

Reply:
`edge` and `mid` are split now in 884ea728cb because GitHub Actions apparently doesn't support single-file bind mounts.

In order to do that, `edge`'s `/opt/init.d/99-run.sh` came from `edge/run.sh` and `mid`'s `/opt/init.d/99-run.sh` came from `mid/run.sh`, so, without attempting to reconcile those 2 scripts, it just meant 2 separate images.

The

```dockerfile
COPY infrastructure/cdn-in-a-box/edge/init.d/ /opt/init.d/
```

layer is essentially at the end of the Dockerfiles anyway, so the other layers are still cached.

Reply:
That all makes sense.